src/main/resources/reference-data/sscs-venues.csv

```diff
@@ -28,7 +28,7 @@
 90,SC090,SSCS Glasgow,St Leonards Church Hall,Woodmill Street,,Dunfermline,,KY11 4JN,,501,"https://www.google.com/maps/place/HM+Courts+%26+Tribunal+Service/@56.0675323,-3.4550886,17z/data=!4m8!1m2!2m1!1sSt+Leonards+Church+Hall+Woodmill+Street+Dunfermline+KY11+4JN!3m4!1s0x0:0x2795c4a07bb58f29!8m2!3d56.0668728!4d-3.4540281"
 91,SC091,SSCS Glasgow,Riverside House,"Floor 2, Riverside House",502 Gorgie Road,Edinburgh,,EH11 3AF,,501,"https://www.google.com/maps/place/Floor+2%2C+Riverside+House+502+Gorgie+Road+Edinburgh+EH11+3AF/@55.932368,-3.250028"
 94,SC094,SSCS Glasgow,Old Gala House,Scott Street,,,,TD1 3JS,,501,"https://www.google.com/maps/place/Old+Gala+House/@55.6139186,-2.8109789,17z/data=!3m1!4b1!4m5!3m4!1s0x48878497ed0146c9:0xb52f68bbad9c92e2!8m2!3d55.6139156!4d-2.8087902"
-96,SC100,SSCS Glasgow,Wellington House,134-136 Wellington Street,,Glasgow,,G2 2XL,0141 353 1441,501,"https://www.google.com/maps/place/134-136+Wellington+Street+Glasgow+G2+2XL/@55.864541,-4.259269"
+96,SC100,SSCS Glasgow,The Glasgow Tribunals Centre,20 York Street,,Glasgow,,G2 8GT,0141 353 1441,501,"https://www.google.com/maps/place/the+glasgow+tribunals+centre+20+york+street+glasgow+g2+8gt/@55.857176,-4.2646086,17z"
```

Comment:
"https://www.google.com/maps/place/the+glasgow+tribunals+centre+20+york+street+glasgow+g2+8gt/@55.857176,-4.2646086,17z" - the url does not work as expected.